config/kubernetes/org.yaml

```diff
@@ -2138,7 +2143,8 @@ teams:
       sample-apiserver: write
       sample-controller: write
   kubernetes-staging-publish-cops:
-    description: will be notified if the robot fails to publish the staging folder to repos
```

Comment:
Maybe drop these changes due to local linters?

Reply:
@palnabarun corrected. Thanks for pointing! 

(Just noticed it was also included in one of my past membership PR too. Will fix this for later!)